src/routes/api/rooms.ts

```diff
@@ -1,10 +1,11 @@
 import { Router } from 'express';
 import queryRoomsRoute from './rooms/queryRooms';
 import createRoomRoute from './rooms/createRoom';
+import authMiddleware from '../../middleware/auth/jwtVerify';
 
 const router = Router();
 
-router.get('/', queryRoomsRoute);
-router.post('/create', createRoomRoute);
+router.get('/', authMiddleware, queryRoomsRoute);
```

Comment:
Thoughts on allowing querying to work for unregistered users and only requiring an account to actually book the room?